fisalert-service/src/main/resources/pipeline-app-build.yml

```diff
@@ -11,7 +11,7 @@ parameters:
   value: https://github.com/RHsyseng/IntegrationApp-Automation.git
   required: true
 - name: GIT_BRANCH
-  value: master
+  value: master 
```

Comment:
What's the change in this file? Just a space at the end?

Reply:
not intended to change. 
It is just that when we do the testing, we need to use feature branch name, and before we make the PR, we need to change the name back to master. A space was there probably get removed by these steps.

Reply:
let me remove it.

Reply:
done